libs/entry-components/styles/modules/components/tables/_rows.scss

```diff
@@ -1,25 +1,48 @@
+/* stylelint-disable max-nesting-depth */
 @use 'sass:map';
-@use '@enigmatry/scss-foundation/src/modules/lists/row-coloring' as list;
+
+@mixin row-coloring($color-map) {
```

Comment:
You don't need this mixin. List utility from Scss Foundation have it already.
We don't need to check whether we have key or not, since default theme will always have some default values.